plugins/woocommerce/includes/class-wc-webhook.php

```diff
@@ -610,10 +610,13 @@ private function failed_delivery() {
 	/**
 	 * Get the delivery logs for this webhook.
 	 *
+	 * @deprecated 8.6.0
 	 * @since  3.3.0
 	 * @return string
 	 */
 	public function get_delivery_logs() {
+		wc_deprecated_function( 'WC_Webhook::get_delivery_logs', '8.6.0' );
```

Comment:
🤔 The weird thing is that this method does not appear to be used anywhere in the codebase. But, since it could theoretically be used by an extension, I decided to undeprecate it and update it to return an accurate URL for the logs, regardless of whether the filesystem or the database is the chosen storage location. a4e81d2c232c76e5cf88ee6e878789e26da722e2